.gitmodules

```diff
@@ -644,3 +644,6 @@
 [submodule "vendor/grammars/smali-sublime"]
 	path = vendor/grammars/smali-sublime
 	url = https://github.com/ShaneWilton/sublime-smali
+[submodule "vendor/grammars/language-ncl"]
+	path = vendor/grammars/language-ncl
+	url = git://github.com/rpavlick/language-ncl.git
```

Comment:
Cloning the submodule fails for me in local with this URL. Could you use `git@` or `https://`?